src/test/java/org/cactoos/io/ReaderOfTest.java

```diff
@@ -40,18 +45,307 @@
  * @checkstyle JavadocMethodCheck (500 lines)
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
+@SuppressWarnings("PMD.TooManyMethods")
 public final class ReaderOfTest {
 
+    /**
+     * Test content for all the tests in this class.
+     */
+    private static final String CONTENT =
+        "Hello, товарищ äÄ üÜ öÖ and ß";
```

Comment:
@proshin-roman it's a bad idea, see http://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html